modules/nextflow/src/main/groovy/nextflow/dag/GexfRenderer.groovy

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright 2013-2019, Centre for Genomic Regulation (CRG)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package nextflow.dag
+import java.nio.charset.Charset
+import java.nio.file.Path
+import java.nio.file.Files
+import java.text.SimpleDateFormat;
+import javax.xml.namespace.QName;
+import javax.xml.stream.XMLEventReader;
+import javax.xml.stream.XMLInputFactory;
+import javax.xml.stream.XMLOutputFactory;
+import javax.xml.stream.XMLResolver;
+import javax.xml.stream.XMLStreamException;
+import javax.xml.stream.XMLStreamWriter;
+import javax.xml.stream.events.Attribute;
+import javax.xml.stream.events.EndElement;
+import javax.xml.stream.events.StartElement;
+import javax.xml.stream.events.XMLEvent;
+
+/**
+ * Render the DAG in Gexf/Gephi
+ * See https://gephi.org/ for more info.
+ *
+ * @author Pierre Lindenbaum / yokofakun
+ */
+class GexfRenderer implements DagRenderer {
+    private static final String XMLNS="http://www.gexf.net/1.3";
+    private static final String XSI_SCHEMA_LOCATION="http://www.gexf.net/1.3 http://www.gexf.net/1.3/gexf.xsd";
+    private static final String VERSION="1.3";
+
+
+    private final String name;
+    GexfRenderer(String name) {
+    this.name = name;    
+    }
+
```

Comment:
There's some problem with the indentation?  maybe tab instead of spaces.. ? https://www.youtube.com/watch?v=SsoOG6ZeyUI 😄